bgpd/bgpd.h

```diff
@@ -203,6 +203,7 @@ struct vpn_policy {
 	/* should be mpls_label_t? */
 	uint32_t tovpn_label; /* may be MPLS_LABEL_NONE */
 	uint32_t tovpn_zebra_vrf_label_last_sent;
+	uint32_t tovpn_zebra_vrf_label_to_send;
```

Comment:
Isn't possible to do the same with `tovpn_zebra_vrf_label_last_sent` variable?

Reply:
actually, If I remove this line, this should be just as you request it.
(it was already moved).

Reply:
`last_sent`  variable is useless now. what I can do is to remove this variable. somehow, we would spare one variable to do this.

Reply:
Indeed, it is not used anymore. You should rename the variable in this commit instead of adding one.
```
$ git grep tovpn_zebra_vrf_label_last_sent
bgpd/bgp_route.c:                               .tovpn_zebra_vrf_label_last_sent =
bgpd/bgpd.c:            bgp->vpn_policy[afi].tovpn_zebra_vrf_label_last_sent =
bgpd/bgpd.h:    uint32_t tovpn_zebra_vrf_label_last_sent;
```

By the way, `tovpn_zebra_vrf_label_to_send` is not inited in bgp_create().

Reply:
added in the initialisation